samples/snippets/create_deny_policy.py

```diff
@@ -99,8 +99,8 @@ def create_deny_policy(project_id: str, policy_id: str) -> None:
     request.policy = policy
     request.policy_id = policy_id
 
-    # Build the create policy request.
-    policies_client.create_policy(request=request)
+    # Build the create policy request and wait for the operation to complete.
+    policies_client.create_policy(request=request).result()
     print(f"Created the deny policy: {policy_id}")
```

Comment:
Rather than printing the `policy_id` from the request, can we capture the result and print its `policy_id`? That would be a more convincing demonstration that the request actually succeeded.

Same comment for the delete and update samples.

Reply:
Done. Thanks.